.github/workflows/branch-merge.yml

```diff
@@ -0,0 +1,33 @@
+name: Auto-merge branches 
+
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  # It was decided to don't run the workflow automatically for now
+  #schedule:
+  #  - cron: '0 0 * * *'
```

Comment:
Theoretically, we could enable the automated cron job for this repository with much less activity. That would avoid the risk of forgetting to trigger it manually.

You would just need to remove the `#` in the two lines above, if you want to do so. Apart from that, it should be good to go 👍

Reply:
Thanks for review. I leave it manual for now, but thats the reason I left the disabled lines in here as well.